src/smart-components/MessageSearch/hooks/useSearchStringEffect.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+
+interface DynamicParams {
+  searchString: string;
+}
+interface StaticParams {
+  setRequestString(string: string): void;
+}
+
+const DEBOUNCING_TIME = 500;
```

Comment:
@sravan-s Do you think it's good to get a debouncing time from the props of MessageSearch component?

Reply:
I can add that prop during modularization